core/src/cpu/air/memory.rs

```diff
@@ -43,6 +43,61 @@ impl CpuChip {
         opcode_selectors.is_sb + opcode_selectors.is_sh + opcode_selectors.is_sw
     }
 
+    /// Constrains the addr_aligned, addr_offset, and addr_word memory columns.
+    ///
+    /// This method will first constrain the addr_word memory column by ensuring it equals to
```

Comment:
This comment here also really doesn't say much / it's too low level (same abstraction level as the code itself), like I think it's better to be like

1) Calculate that the unaligned address is correctly computed to be `op_b.value + op_c.value`.
2) Calculate that the address offset is address % 4.
3) Assert the validity of the aligned address given the address offset and the unaligned address.